# Add easybuild locations of dependencies
# December 13th 2018 by B. Hajgato (Free University Brussels - VUB)
# updated for 2.11.4 by Jasper Grimm (University of York)
diff -Nru libxml2-2.11.4.orig/python/setup.py.in libxml2-2.11.4/python/setup.py.in
--- libxml2-2.11.4.orig/python/setup.py.in	2024-01-23 14:42:40.321205584 +0000
+++ libxml2-2.11.4/python/setup.py.in	2024-01-23 14:54:31.558212220 +0000
@@ -71,11 +71,8 @@
 # - libxml2/libxml/tree.h
 # - libxslt/xsltconfig.h
 includes_dir = [
-"/usr/include",
-"/usr/local/include",
-"/opt/include",
-os.path.join(ROOT,'include'),
-HOME
+    os.path.join(os.getenv("EBROOTLIBXML2"),"include"),
+    os.path.join(os.getenv("EBROOTLIBICONV"),"include"),
 ];
 
 xml_includes=""
